src/library/Uno.Themes.WinUI.Markup/Theme/PipsPager.cs

```diff
@@ -0,0 +1,247 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+using Windows.Devices.Bluetooth.Advertisement;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class PipsPager
+		{
+			public static class Resources
+			{
+				public static class Default
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorForeground")]
```

Comment:
Should these move to a `SelectionIndicator` class? @kazo0

Reply:
Yes, these should all be inside of a SelectionIndicator class for both Foreground and Background.

`PipsPager.Resources.SelectionIndicator.Foreground.Default/PointerOver/Pressed/etc.`
`PipsPager.Resources.SelectionIndicator.Background.Default/PointerOver/Pressed/etc.`

Reply:
I put SelectionIndicatorButton